### ISSUE TITLE
Retrieve AddDevice result

### ISSUE THREAD

**Issue by bvdzwet** [OSGP/Platform#218]
As a grid operator, after a request to AddDevice from the SmartMeteringInstallation service, I want to be able to retrieve the result by issuing an AsyncRequest using the correlation UID from the initial call.

**Comment by rudidevries** [comment]
### User story
As a Grid Operator,
I want to  retrieve the result for an addDevice request from the SmartMeteringInstallation service by issuing an AsyncRequest,
so I have a consistent way of retrieving the results for asynchronous calls
 
### Scenarios
 
#### Retrieve result
Given I have sent an addDevice request to the SmartMeteringInstallation 
When the device has been added
Then I get a notification containing the Correlation UID
and I can retrieve the result by issuing an AsyncRequest using the Correlation UID
 
#### Removal of result
Given a device has been added for an addDevice request
When I have retrieved the result for the addDevice call
Then the result of the addDevice call is no longer stored in the platform

**Comment by rlemmers** [comment]
@elhachhouchi should this be in the current sprint or not? @rudidevries picked it up, because there was no other work at that time.

**Comment by rudidevries** [comment]
I have worked in a branch called "feat-retrieve-device-result".  I have added changes to store the result type in the database, and added this to two of the webservice methods. So this story is not completed, but the changes could be merged without problems.

I have merged the pull request of "feat-smartmeter-set-tariff" because this has some overlapping changes. As this pull request is not yet merged, I will not create a pull request at this time. I will merge with development later.
